src/imap.rs

```diff
@@ -802,7 +804,7 @@ impl Imap {
         }
 
         info!(context, "Done fetching existing messages.");
-        Ok(())
+        Ok(created)
```

Comment:
Can we add the device message right from this function? Otherwise it's strange that the function returns smth very particular, it's about messages, but returns the number of contacts